core/platform/search/elastic_search_services.py

```diff
@@ -0,0 +1,167 @@
+# coding: utf-8
+#
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS-IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Provides platform search services implemented using the elastic search python
+API.
+"""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import json
+
+import feconf
+import python_utils
+
+import elasticsearch
+
+# https://www.elastic.co/guide/en/elasticsearch/reference/current/index-modules.html#index-max-result-window
+# This is the maximum number of pages to page through using the from and size
+# parameters to page through any given index. This number is equal to the limit
+# index.max_result_window and defines the maximum the sum of the parameters
+# (size + from) can be. If needed, we will have to change this to use the
+# search_after parameter or scrolling option as mentioned in the link above.
+MAXIMUM_NUMBER_OF_PAGES = 10000
+ES = elasticsearch.Elasticsearch()
+
+
+def add_documents_to_index(documents, index_name):
+    """Adds a document to an index.
+
+    Args:
+        documents: list(dict). Each document should be a dictionary.
+            Every key in the document is a field name, and the corresponding
+            value will be the field's value.
```

Comment:
Nit: please combine all this into a single paragraph of text (i.e. don't artificially add a line break after "dictionary." and "value.").